play-testkit/src/test/scala/example/myapp/helloworld/grpc/actions/helloworld/GreeterServiceWithActionsImpl.scala

```diff
@@ -21,8 +21,6 @@ class GreeterServiceWithActionsImpl @Inject() (
     actionBuilder: DefaultActionBuilder,
 ) extends AbstractGreeterServiceRouter(
       actorSystem,
-      parsers,
-      actionBuilder,
```

Comment:
If they don't need to be passed to AbstractGreeterServiceRouter anymore, maybe they don't need to be in the parameters for this class at all anymore?